resources/console/charts/web/templates/deployment.yaml

```diff
@@ -35,6 +35,8 @@ spec:
         - name: console
           image: "{{ .Values.global.containerRegistry.path }}/console:{{ .Values.console.image.tag }}"
           imagePullPolicy: {{ .Values.console.image.pullPolicy }}
+          resources: 
+            {{- toYaml .Values.deployment.resources | nindent 12 }}
```

Comment:
what about the `core-ui` container? And other "frontend" deployments like `log-ui` or `service-catalog-io`?